src/Razor/test/RazorLanguage.Test/TestFiles/IntegrationTests/ComponentDesignTimeCodeGenerationTest/Component_WithKey_WithChildContent/TestComponent.ir.txt

```diff
@@ -0,0 +1,31 @@
+Document - 
+    NamespaceDeclaration -  - Test
+        UsingDirective - (3:1,1 [12] ) - System
+        UsingDirective - (18:2,1 [32] ) - System.Collections.Generic
+        UsingDirective - (53:3,1 [17] ) - System.Linq
+        UsingDirective - (73:4,1 [28] ) - System.Threading.Tasks
+        UsingDirective - (104:5,1 [37] ) - Microsoft.AspNetCore.Components
+        ClassDeclaration -  - public - TestComponent - Microsoft.AspNetCore.Components.ComponentBase - 
+            DesignTimeDirective - 
+            CSharpCode - 
+                IntermediateToken -  - CSharp - #pragma warning disable 0414
+            CSharpCode - 
+                IntermediateToken -  - CSharp - private static System.Object __o = null;
+            CSharpCode - 
+                IntermediateToken -  - CSharp - #pragma warning restore 0414
+            MethodDeclaration -  - protected override - void - BuildRenderTree
+                Component - (0:0,0 [95] x:\dir\subdir\Test\TestComponent.cshtml) - MyComponent
+                    ComponentChildContent -  - ChildContent - context
+                        HtmlContent - (44:0,44 [11] x:\dir\subdir\Test\TestComponent.cshtml)
+                            IntermediateToken - (44:0,44 [11] x:\dir\subdir\Test\TestComponent.cshtml) - Html - \n    Some 
+                        MarkupElement - (55:1,9 [16] x:\dir\subdir\Test\TestComponent.cshtml) - el
+                            HtmlContent - (59:1,13 [7] x:\dir\subdir\Test\TestComponent.cshtml)
+                                IntermediateToken - (59:1,13 [7] x:\dir\subdir\Test\TestComponent.cshtml) - Html - further
+                        HtmlContent - (71:1,25 [10] x:\dir\subdir\Test\TestComponent.cshtml)
+                            IntermediateToken - (71:1,25 [10] x:\dir\subdir\Test\TestComponent.cshtml) - Html -  content\n
+                    SetKey - (18:0,18 [9] x:\dir\subdir\Test\TestComponent.cshtml)
```

Comment:
It looks like your `FormatNode` implementation isn't working. I would expect to see the content in here. We want that to show up in the diff so we can test that it doesn't change.

Reply:
Your `FormatNode` looks right, so I'm guessing you added it generating these baselines.

Reply:
No, the baselines were generated last. Not sure why `FormatNode` isn't doing the right thing. I'll investigate.

Reply:
Turns I had missed an override on `IntermediateNodeWriter` so my `FormatNode` here was never being called. Fixed.